src/registrar/templates/emails/domain_invitation.txt

```diff
@@ -1,6 +1,25 @@
-You have been invited to manage the domain {{ domain.name }} on get.gov, 
-the registrar for .gov domain names.
+{% autoescape off %}{# In a text file, we don't want to have HTML entities escaped #}
+Hi.
 
-To accept your invitation, go to <{{ domain_url }}>.
+{{ full_name }} has added you as a manager on {{ domain.name }}.
 
-You will need to log in with a Login.gov account using this email address.
+YOU NEED A LOGIN.GOV ACCOUNT
+You’ll need a Login.gov account to manage your .gov domain. Login.gov provides a simple and secure process for signing into many government services with one account. If you don’t already have one, follow these steps to create your Login.gov account <https://login.gov/help/get-started/create-your-account/>. 
```

Comment:
**comment (non-blocking)**: Because these are plain text emails, we don't have control over the line wrapping that people will see in their email client if we are using these single long lines for each paragraph. I think that it might be good to wrap these text files at something like 60-80 columns wide so that we can ensure shorter lines and correspondingly better comprehension.

Reply:
TIL, thank you for this feedback!!!